scripts/check-coverage.jsh

```diff
@@ -51,11 +51,11 @@ var moduleCoverageData = List.of(
   new ModuleCoverage("sirius-components-collaborative-forms", 92.0),
   new ModuleCoverage("sirius-components-forms-graphql", 83.0),
   new ModuleCoverage("sirius-components-widget-reference", 83.0),
-  new ModuleCoverage("sirius-components-collaborative-widget-reference", 72.0),
-  new ModuleCoverage("sirius-components-widget-reference-graphql", 52.0),
+  new ModuleCoverage("sirius-components-collaborative-widget-reference", 71.0),
+  new ModuleCoverage("sirius-components-widget-reference-graphql", 51.0),
```

Comment:
Why?

Reply:
So why does the new version has a lower code coverage?

Reply:
https://github.com/eclipse-sirius/sirius-web/pull/3977#issuecomment-2356011989